source/codegen/templates/service_helpers.mako

```diff
@@ -197,6 +199,11 @@ ${initialize_standard_input_param(function_name, parameter)}\
       ${parameter['type']} ${parameter_name} = request->${field_name}().size();\
 </%def>
 
+## Initialize a 'pass_null' param.
+<%def name="initialize_pass_null_param(parameter)">\
+      auto ${parameter["name"]} = nullptr;\
```

Comment:
I think you'll have to do the same `common_helpers.camel_to_snake(parameter['cppName'])` other initialize methods do to come up with the variable name. This will make sure it aligns with what's expected later when we call `CreateArgs`. I think eventually this could be cleaned up and possibly refactored into the metadata_mutation stuff but for now just doing it in this method should be fine.